spring-core/src/main/java/org/springframework/util/backoff/ExponentialBackOff.java

```diff
@@ -160,11 +168,33 @@ public void setMaxElapsedTime(long maxElapsedTime) {
 	/**
 	 * Return the maximum elapsed time in milliseconds after which a call to
 	 * {@link BackOffExecution#nextBackOff()} returns {@link BackOffExecution#STOP}.
+	 * @return the maxElapsedTime.
+	 * @see #getMaxAttempts()
 	 */
 	public long getMaxElapsedTime() {
 		return this.maxElapsedTime;
 	}
 
+	/**
+	 * The maximum number of attempts after which a call to
+	 * {@link BackOffExecution#nextBackOff()} returns {@link BackOffExecution#STOP}.
+	 * @param maxAttempts the maxAttempts.
+	 * @see #setMaxElapsedTime(long)
```

Comment:
Probably some `@since` has to be here on public API.
Let's start from the current one and we'll see how this PR going to be approved!